presto-docs/src/main/sphinx/release/release-0.219.rst

```diff
@@ -0,0 +1,62 @@
+* Improve result comparison for orderable array columns by applying :func:`array_sort` before :func:`checksum`.
+* Improve result comparison for ``Select`` queries by rewriting ``Select`` queries as ``CreateTableAsSelect`` and using checksum queries to verify the results. This eliminates the row count limit for Select queries.
+* Improve Verifier interfaces to allow easy extension by community users.
+* Reuse initial control query results for the determinism check. This reduces the maximum number of control query runs from 3 to 2 and also eliminates test query reruns.
+* Add support for retrying transient Presto query failures, using configuration properties ``presto.max-attempts``, ``presto.min-backoff-delay``, ``presto.max-backoff-delay``, ``presto.backoff-scale-factor``. Similar configurations prefixed with ``cluster-connection`` instead of ``presto`` are introduced for retrying transient network failures when talking to Presto server. Intermediate failures are recorded and emitted in the output.
+* Add support for automatically resolving certain kinds of failures including exceeding the global memory limit and exceeding time limit.
+* Add configuration properties ``metadata-timeout`` and ``checksum-timeout`` to set the timeouts for metadata queries (i.e. describe queries that reads table schema) and checksum queries.
+* Add configuration property ``source-query.table-name`` to allow the table name of the MySQL table from which verifier queries are loaded to be configured.
+* Add configuration property ``human-readable.log-file`` to allow human-readable events to be logged into the specified file instead of to ``stdout``.
+* Rename configuration property ``query-database`` to ``source-query.database``.
+* Rename configuration property ``suites`` to ``source-query.suites``.
+* Rename configuration property ``max_queries`` to ``source-query.max-queries-per-suite``.
+* Rename configuration property ``event-client`` to ``event-clients``.
+* Rename configuration property ``event-log-file`` to ``json.log-file``.
+* Rename configuration property ``run-id`` to ``test-id``.
+* Rename configuration property ``thread-count`` to ``max-concurrency``.
+* Rename configuration properties ``control-gateway`` to ``control.jdbc_url``, and ``test-gateway`` to ``test.jdbc_url``.
+* Rename configuration properties ``shadow-writes.control-table-prefix`` to ``control.table-prefix``, and ``shadow-writes.test-table-prefix`` to ``test.table-prefix``.
+* Remove configuration properties ``control.query-types``, ``test.query-types``, ``source``, ``max-row-count``, ``always-report``, ``skip-correctness-regex ``, ``check-correctness ``, ``skip-cpu-check-regex ``, ``check-cpu``, ``explain-only``, ``verbose-results-comparison``, ``quiet``, ``control-teardown-retries``, ``test-teardown-retries``, ``shadow-writes``.
+
+SPI Changes
+-----------
+* Expose ``RowExpression`` to SPI.
```

Comment:
- Just to confirm, can this break any connector out there? If yes, we should add a warning. 
- Also we may need to give a bit more info here: what's the benefit, use case, etc.

Reply:
@hellium01

Reply:
It shouldn't break any connector, RowExpression is previously only used internally, we wish to start using it to pass resolved expressions to connectors.

Reply:
Changed to 
```
Expose ``RowExpression`` to SPI, to allow passing resolved expressions to connectors.
```